### ISSUE TITLE
'Image-width' Less function doesn't work

### ISSUE THREAD

**Issue by NelGarbuzova** [oklai/koala#728]
**Koala version 2.3.0**

**Bug report or feature request?**
No bag report. 

image-width("file.png") less function doesn't work. It's just write text 'image-width("file.png")'. But when I use less compiler in terminal, it works great. Does this mean Koala does not support all 2.2.0 less functions?

**Comment by zaygraveyard** [comment]
Yes Koala ships with LESS v1.7.5, but this will change soon (if find the time to update Koala)
